ledger/ledger-utxo/src/main/kotlin/net/corda/v5/ledger/utxo/UtxoLedgerService.kt

```diff
@@ -120,4 +121,79 @@ interface UtxoLedgerService {
         session: FlowSession,
         validator: UtxoTransactionValidator
     ): UtxoSignedTransaction
+
+    /**
+     * Sends a transaction builder to another session with all dependent back chains.
+     *
+     * @param transactionBuilder The [UtxoTransactionBuilder] to send.
+     * @param session The receiver [FlowSession].
+     */
+    @Suspendable
+    fun sendTransactionBuilder(
+        transactionBuilder: UtxoTransactionBuilder,
+        session: FlowSession
+    )
+
+    /**
+     * Receives a transaction builder from another session.
+     *
+     * @param session The [FlowSession] to receive the [UtxoTransactionBuilder] from.
+     */
+    @Suspendable
+    fun receiveTransactionBuilder(
+        session: FlowSession,
+    ): UtxoTransactionBuilder
```

Comment:
These functions are just syntactic sugar over the diff ones below - I wonder whether they have to be on the API or could be just static helper functions? Maybe we could build the whole dealing with receiving/sending as helper functions, if we only have the diff-sending/receiving functions on the API?

Reply:
I have removed the discarded options.